lib/std/process/Child.zig

```diff
@@ -1389,7 +1389,8 @@ fn windowsMakeAsyncPipe(rd: *?windows.HANDLE, wr: *?windows.HANDLE, sattr: *cons
 
 var pipe_name_counter = std.atomic.Value(u32).init(1);
 
-// Should be kept in sync with `windowsCreateProcessSupportsExtension`
+// Should be kept in sync with `windowsCreateProcessSupportsExtension` and
+// `std.Build.supportedWindowsProgramExtension()`.
```

Comment:
Let's rename `CreateProcessSupportedExtension` to `WindowsExtension` and make it `pub`. Then you can iterate over the enum tags in the build system rather than having a "keep in sync" comment.